src/build.ts

```diff
@@ -73,9 +74,20 @@ export const get_svelte_files = async ({
 	return files;
 };
 
-const copy_assets = (
-	{ site_dir, out_dir }: Partial<Options>,
-) => copy(site_dir + "assets", out_dir + "assets", { overwrite: true });
+const copy_assets = async (
+	{ site_dir, out_dir }: Pick<Options, "site_dir" | "out_dir">,
+) => {
+	for await (
+		const file of expandGlob("**/*.!(svelte)", { root: site_dir })
+	) {
+		const out_file = file.path.replace(site_dir, out_dir);
+
+		await ensureDir(dirname(out_file));
+		await copy(file.path, out_file, {
+			overwrite: true,
+		});
```

Comment:
Could this lead to overwriting a file that has the same name as one of our outputs–e.g. `index.html` or `islands.js`? Might be worth ensuring that there’s no possible conflict.

Reply:
yeah it could. what do you reckon is the best way to handle that? work out if it would and error with an explanation?

Reply:
Maybe by not overwriting the file and providing a good error:

```
await copy(file.path, out_file).catch(() => `The file at ${file.path} conflicts with an existing file. Not copied.`)
```ts